_includes/support_list.html

```diff
@@ -18,7 +18,7 @@
 {% assign sw_index = 0 %}
 {% assign sw_index_body = 0 %}
 {% assign sw_count = 0 %}
-{% assign max_sw_per_table = 9 %}
+{% assign max_sw_per_table = 7 %}
```

Comment:
Can you also update `min_fullwidth_display` in the next line to be `6`?

Reply:
Good catch, that's much better.